os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -253,6 +258,7 @@ void rtl8730e_mipidsi_underflowreset(void)
 		lcddbg("ERROR: LCDC_CTRL 0x%x\n", pLCDC->LCDC_CTRL);
 
 		rtl8730e_enable_lcdc();
+		mipidsi_mode_switch(true);
```

Comment:
Regardless of this PR, `mipi mode switch true` is difficult to understand in code readability. Could you rename it with another commit?

Reply:
mipidsi_mode_switch(true) --> video mode
mipidsi_mode_switch(false) --> cmd mode
this has been taken care below, maybe i will just change it to call rtl8730e_mipi_mode_switch instead of rename mipidsi_mode_switch? 
static void rtl8730e_mipi_mode_switch(mipi_mode_t mode)
{
	if (mode == CMD_MODE) {
		mipidsi_mode_switch(false);
		MIPI_DSI_INT_Config(MIPI, DISABLE, ENABLE, FALSE);
		DelayMs(140);
	} else {
		MIPI_DSI_INT_Config(MIPI, DISABLE, DISABLE, FALSE);
		mipidsi_mode_switch(true);
	}
}

Reply:
mipi_mode_switch_to_video(true)
mipi_mode_switch_to_video(false)

or

mipi_mode_switch(VIDEO_MODE)
mipi_mode_switch(CMD_MODE)

I suggest renaming like above.
Current code is impossible to change what. I should go into the function and should check code again.

Reply:
Hi @sunghan-chang, new commit to rename done.